CHANGELOG.md

```diff
@@ -14,7 +14,13 @@
 
 -->
 
-## 1.4.6
+### Master
+
+## 1.4.7
```

Comment:
This needs a triple `#` for emission-nebula. We changed this recently, the current code is here: https://github.com/artsy/emission-nebula/blob/904321ac7a8becb79a91b240a09233c416dbd959/fastlane/Fastfile#L63

I will send a Danger rule requiring this.